app/views/index.html

```diff
@@ -26,9 +26,25 @@ <h1>Urgent Care Signposting Prototypes</h1>
 
                   <h2>Prototypes</h2>
                   <ul>
-                    <li><a href="/walk-in-finder">The interrupter - Walk in finder</a></li>
-                    <li><a href="/google/search">Google search, the interrupter & Urgent Care Pete for A&E finder</a></li>
+                    <li>
+                    <p>Interruptor</p>
+                        <ol>
+                            <li><a href="/google/search-wi">Google journey Walk-in</a></li>
+                            <li><a href="/walk-in-finder">Direct to Walk-in finder</a></li>
+                        </ol>
+                    </li>
+                    <li>
+                    <p>Interruptor & Pete</p>
+                        <ol>
+                            <li><a href="/google/search-ae">Google journey A&E</a></li>
+                            <li><a href="/ae-finder">Direct to A&E finder</a></li>
+                        </ol>
+                    </li> 
                     <li><a href="/carecarder">Care Carder</a> </li>
+                    <li><a href="https://www.nhs.uk/conditions/diabetic-eye-screening-approach-1/">Diabetic eye screening test approach 1</a></li>
```

Comment:
What are these links for?

Reply:
Mark and Ana asked me to add them to the index so they could have all the things they're testing in one place